test/integration/cart_integration_test.rb

```diff
@@ -24,64 +24,64 @@ class CartIntegrationTest < ActionDispatch::IntegrationTest
     end
   end
 
-  test 'an unauthorized user with items in their cart can view their cart' do
-    item = Item.create(title: 'coffee', description: 'black nectar of the gods', price: 1200)
-    visit "/items/#{item.id}"
+  test 'an unauthorized user with projects in their cart can view their cart' do
+    project = Project.create(title: 'coffee', description: 'black nectar of the gods', price: 1200)
```

Comment:
Line is too long. [99/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.